check_api/pom.xml

```diff
@@ -197,6 +213,17 @@
         </executions>
 
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Multi-Release>true</Multi-Release>
+            </manifestEntries>
+          </archive>
+        </configuration>
+      </plugin>
```

Comment:
Also had to add this to make sure that the class file bundled under `META-INF/versions/23` is picked up.